### PR TITLE
feat: update banner bg of blog page

### PR DESCRIPTION
**Pull request by Keith-CY**
Use hi-res png provided by designer

https://user-images.githubusercontent.com/7271329/226196185-9f838ecb-2620-4204-a8f4-3808761c62f9.mov

### CODE REVIEW

File: src/pages/blogs/index.page.tsx [hunk 1]
```diff
@@ -45,9 +45,9 @@ const Index = ({ blogs, populars, categories, pageCount }: Props) => {
       <Page>
         <div className={styles.banner}>
           <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerRight} alt="cell" className={styles.right} />
+          {/* <Image src={blogBannerLeft} alt="cell" className={styles.left} /> */}
           <Image src={blogBannerRight} alt="cell" className={styles.right} />
+          {/* <Image src={blogBannerRight} alt="cell" className={styles.right} /> */}
```
Comment (WhiteMinds) [inline src/pages/blogs/index.page.tsx:50]: need to be removed

**Review by WhiteMinds — CHANGES_REQUESTED** [review]

File: src/pages/blogs/index.page.tsx [hunk 2]
```diff
@@ -45,9 +45,9 @@ const Index = ({ blogs, populars, categories, pageCount }: Props) => {
       <Page>
         <div className={styles.banner}>
           <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerLeft} alt="cell" className={styles.left} />
-          <Image src={blogBannerRight} alt="cell" className={styles.right} />
+          {/* <Image src={blogBannerLeft} alt="cell" className={styles.left} /> */}
```
Comment (Keith-CY) [inline src/pages/blogs/index.page.tsx:48]:
```suggestion
```

File: src/pages/blogs/index.page.tsx (hunk 1 above)
Comment (Keith-CY) [reply src/pages/blogs/index.page.tsx:50]:
```suggestion
```

Comment (Keith-CY) [reply src/pages/blogs/index.page.tsx]: Updated by https://github.com/Magickbase/nervos-official-website/pull/36/commits/e8e5e0c721e5a95c8a47431c3521b961b308b633

**Review by WhiteMinds — APPROVED** [review]

**Pull request merged**